qiskit_optimization/translators/ising.py

```diff
@@ -184,9 +184,9 @@ def from_ising(
         warn(
             "The `qubit_op` argument can currently accept Opflow operators (`OperatorBase` type), "
             "but have been superseded by Qiskit Terra quantum_info `BaseOperators` such as "
-            "`SparsePauliOp`. Opflow operator support will be deprecated in a future release "
+            "`SparsePauliOp`. Opflow operator has been deprecated in Qiskit Terra 0.25.0 "
             "and subsequently removed after that.",
-            category=PendingDeprecationWarning,
+            category=DeprecationWarning,
```

Comment:
We should likewise note this in a reno deprecations section,

Reply:
done